src/metpy/calc/tools.py

```diff
@@ -825,6 +825,47 @@ def lat_lon_grid_deltas(longitude, latitude, y_dim=-2, x_dim=-1, **kwargs):
     return dx * units.meter, dy * units.meter
 
 
+@exporter.export
+@preprocess_xarray
+def azimuth_range_to_lat_lon(azimuths, ranges, center_lon, center_lat, **kwargs):
+    """Convert azimuth and range bins in a polar coordinate system to lat/lon coordinates.
+
+    Pole refers to the origin of the coordinate system. Adapted from Brian Blaylock's code.
```

Comment:
Could we move the credit to maybe a `Notes` section in the docstring as something like 'Credit to Brian Blaylock for the original implementation.' ?